homework.rb

```diff
@@ -1,6 +1,20 @@
-VARIANT = 0
+VARIANT = 2
 
 def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+	"#{digits}-#{letters}-#{digits}"
 end
+
+def digits
+	rand(1000..9999).to_s
+end
+
+def letters
+	letters, letter, i = ("A".."Z").to_a, "", 0
+
+	while i < 4 do
+		i += 1
+		letter += letters[rand letters.length]
```

Comment:
in this context better to use `rand` with brackets or assign random number to another variable before this line